drivers/NFSSR.py

```diff
@@ -118,9 +121,14 @@ def check_server(self):
 
     def mount(self, mountpoint, remotepath, timeout = 0):
         try:
-            nfs.soft_mount(
-                mountpoint, self.remoteserver, remotepath, self.transport,
-                timeout=timeout, nfsversion=self.nfsversion)
+            if self.srcmd.params.get('device_config').has_key(PROBEVERSION):
```

Comment:
Can we check self.dconf? That should already be populated with all device_config parameters.